web/src/app/components/modal/modal.component.ts

```diff
@@ -306,11 +313,13 @@ export class ModalComponent extends BaseModal implements OnInit {
     }
   }
 
-  validateCasedetails(){
-    
+  validateCasedetails(){   
       this.caseDetailsIsSelected = this.validateDefendantRace.value[0] == ''; 
       this.genderIsSelected = this.validateDefendantGender.value[0] == '';
-      
-  
-}
+  }
+
+  private isFormControlInvalid(formControlName: string): boolean {
+    return this.defendantAndCaseForm.get(formControlName).touched && 
+      this.defendantAndCaseForm.get(formControlName).invalid;
+  }
```

Comment:
In order to bind to the invalid attribute in the ibm component we need to check if the form control is valid. 
We also need to check that it has been touched or else it will show a validation error on load.